terraform/modules/router/route53.tf

```diff
@@ -1,118 +1,124 @@
+provider "aws" {
+  region  = "eu-west-1"
+  version = "~> 2.70"
+}
+
 resource "aws_route53_zone" "root" {
-  name = "${var.domain}"
+  name = var.domain
 }
 
 resource "aws_route53_record" "www_acm_validation" {
-  zone_id = "${aws_route53_zone.root.zone_id}"
-  name    = "${var.www_acm_name}${var.domain}"
+  zone_id = aws_route53_zone.root.zone_id
+  name    = var.www_acm_name}${var.domain
```

Comment:
This looks a little suspect...

Reply:
Yes - I think this may be the cause of the error you were seeing for Preview. We'll take a look.